app/Controllers/CoverController.php

```
@@ -33,24 +31,18 @@ public function download(Request $request, Response $response): Response
         $data = json_decode($input, true);
-        
+
         // If JSON decode failed, try to get parsed body
         if ($data === null) {
-            $data = (array)$request->getParsedBody();
+            $data = (array) $request->getParsedBody();
         }
-        
+
         $coverUrl = trim($data['cover_url'] ?? '');
         $token = $data['csrf_token'] ?? null;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$token'. (undefined)

(UnusedLocalVariable)